doc/source/myrocks/variables.rst

```diff
@@ -2566,6 +2571,17 @@ Specifies whether to use adaptive mutex
 which spins in user space before resorting to the kernel.
 Disabled by default.
 
+.. variable:: rocksdb_use_default_sk_cf
+
+  :version 5.7.32-35: Implemented
```

Comment:
It's not merged yet.
Still waiting for review at https://github.com/percona/percona-server/pull/4081